### PR TITLE
GODRIVER-418: Support nameOnly option in listCollections command

### PR DESCRIPTION
**Pull request by igorKryvenko**

### CODE REVIEW

File: mongo/database.go [hunk 1]
```diff
@@ -90,3 +91,21 @@ func (db *Database) Drop(ctx context.Context) error {
 	}
 	return nil
 }
+
+// ListCollections list collections from mongodb database.
+func (db *Database) ListCollections(ctx context.Context, options []options.ListCollectionsOptioner, filter *bson.Document) (command.Cursor, error) {
```
Comment (skriptble) [inline mongo/database.go]: Let's swap the options and filter arguments and make options variadic so users can choose if they want to provide options.

**Comment by igorKryvenko** [comment]
@skriptble Hi Kris. Is the PR good to merge?

**Comment by skriptble** [comment]
Hi @igorKryvenko, I haven't had a chance to review this against the specification. I'll be able to take a look later today or tomorrow. Sorry for the delay.

Here's the spec if you want to refer to it: https://github.com/mongodb/specifications/blob/master/source/enumerate-collections.rst

**Comment by igorKryvenko** [comment]
@skriptble Kris, np =) I just ask, no more. It is not so important.

**Comment by igorKryvenko** [comment]
@skriptble Hi. Thanks for the response. Can I help you with new option design or it's implementation?

**Comment by skriptble** [comment]
@igorKryvenko, we're currently working through the design doc. If you want to help out another pair of eyes on it would be useful: https://github.com/mongodb/mongo-go-driver/pull/61
Feedback would be appreciated, we're trying to make it an approachable design to any maintainer.

**Comment by skriptble** [comment]
Hi @igorKryvenko, this looks like it covers what's required for [GODRIVER-418](https://jira.mongodb.org/browse/GODRIVER-418) so I'll merge this. Thanks for the contribution!

**Pull request merged**